blogposts/2021/release-3.24.md

```diff
@@ -0,0 +1,31 @@
+---
+title: "Sourcegraph 3.24 release"
+publishDate: 2021-01-20T10:00-07:00
+description: "Sourcegraph 3.24 is released."
+tags: [blog, release]
+slug: "release/3.24"
+published: true
+changelogItems:
+  - description: Added panels to the Sourcegraph monitoring dashboards
```

Comment:
This one feels a bit vague, I think if we can't fit the details in the changelog (new dashboards reference, updated panels, version/alert annotations) it would be best to just omit this entry 😄 (speaking as the person who wrote this entry in the changelog)

Reply:
@aileenrose are we limited to a certain number of characters for this post type? 

@bobheadxi do you have a suggestion for the description that shares more of the details? If we don't have an imposed word limit, we can share whatever is the most valuable, helpful, and informative for the user and reader of the post.

Reply:
```suggestion
  - description: Monitoring dashboards now include links to relevant documentation and annotation overlays for alerts and version changes.
```